sound/soc/intel/common/soc-acpi-intel-ptl-match.c

```diff
@@ -69,6 +101,24 @@ static const struct snd_soc_acpi_adr_device rt711_sdca_0_adr[] = {
 	}
 };
 
+static const struct snd_soc_acpi_adr_device rt712_vb_3_group1_adr[] = {
+	{
+		.adr = 0x000330025D071201ull,
+		.num_endpoints = ARRAY_SIZE(jack_amp_g1_dmic_endpoints),
+		.endpoints = jack_amp_g1_dmic_endpoints,
+		.name_prefix = "rt712"
+	}
+};
+
+static const struct snd_soc_acpi_adr_device rt1320_3_group1_adr[] = {
+	{
+		.adr = 0x000330025D132001ull,
+		.num_endpoints = 1,
+		.endpoints = &spk_r_endpoint,
+		.name_prefix = "rt1320-1"
```

Comment:
is there a need for the '-1' suffix?

Reply:
> is there a need for the '-1' suffix?
yes, we checked here. https://github.com/thesofproject/linux/blob/c2fd38c76c0ac19946a62349fd53a45948a43181/sound/soc/sdw_utils/soc_sdw_rt_amp.c#L205

Reply:
> is there a need for the '-1' suffix?
yes, we checked the substring  at https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sdw_utils/soc_sdw_rt_amp.c#L205